src/components/Comments.jsx

```diff
@@ -1,12 +1,22 @@
 import { useState, useEffect } from "react";
+import { getComments } from '../api';
 
 
-export const Comments = () => {
+export const Comments = ({ review_id }) => {
     const [comments, setComments] = useState([]);
 
     useEffect(() => {
-        getComment().then((comments) => {
-            res.data.comments
+        getComments(review_id).then((comments) => {
+            setComments(comments);
         })
-    })
-}
\ No newline at end of file
+    }, [])
+
+    return (comments.map((comment) => ( <div key={comment.comment_id} className="Comments">
+                <p> <strong>{comment.author}: </strong> {comment.body}</p>
+                <p> <strong>Created at: </strong> {comment.created_at}</p>
+                <p> <strong>Votes: </strong> {comment.votes}</p>
+            </div>
```

Comment:
Is a div the correct tag to use here? Is there any semantic HTML tags which would be more appropiate?